source/hooks/intercept.h

```diff
@@ -253,7 +279,7 @@ struct InterceptDescriptor
 #define MAKE_INTERCEPT(enabled, dll, name) {#dll".dll", #name, (FARPROC)My##name, (FARPROC)Tramp##name, enabled, false}
 #define MAKE_INTERCEPT2(enabled, dll, name, myname) {#dll".dll", #name, (FARPROC)My##myname, (FARPROC)Tramp##myname, enabled, false}
 #define MAKE_INTERCEPT3(enabled, dllWithExt, name, suffix) {#dllWithExt, #name, (FARPROC)My##name##_##suffix, (FARPROC)Tramp##name##_##suffix, enabled, false}
-#define MAKE_INTERCEPT_ORD(enabled, dll, name, ordinal) {#dll".dll", reinterpret_cast<const char*>(ordinal), (FARPROC)My##name, (FARPROC)Tramp##name, enabled, true}
+#define MAKE_INTERCEPT_ORD(enabled, dll, name, ordinal) {#dll".dll", reinterpret_cast<const char*>(ordinal), (FARPROC)My##name, (FARPROC)Tramp##Ordinal##ordinal##_##name, enabled, true}

```

Comment:
This is a quick hack to support the new interface until the new intercept method is completed.